crawl-ref/source/dat/database/randname.txt

```diff
@@ -700,7 +750,7 @@ w:5
 "Gaze Upon Infinity"
 
 w:5
-"I Knew You'd Do That"
+"I Foresaw This"
```

Comment:
Why the change to this one, instead of both?

Reply:
i didn't see there was another one. this one just felt a little odd and ... not *deific* enough for an Ash name. but regular randarts get to be very silly, so maybe it's fine for a few ash artis to border on silliness too?